manifests/server.pp

```diff
@@ -88,6 +88,7 @@
                   $remove_default_accounts = false,
                   $restart                 = $mysql::params::restart,
                   $root_group              = $mysql::params::root_group,
+                  $managed_dirs            = undef,
```

Comment:
$managed_dirs            = $mysql::params::managed_dirs
Could you please add the variable to params file and replace with this line above. Thank you
https://github.com/puppetlabs/puppetlabs-mysql/blob/master/manifests/params.pp

Reply:
@sheenaajay the file https://github.com/puppetlabs/puppetlabs-mysql/blob/master/manifests/params.pp already contains the variable with different values which related with cases.
As I understand you'd like to change a value for the parameter in the class mysql::server
please see my latest commit https://github.com/puppetlabs/puppetlabs-mysql/pull/1305/commits/a22132740b0da0b24eabf70a46dc0db508a849b2